sources/ElkArte/Controller/Unread.php

```diff
@@ -334,20 +334,7 @@ public function action_unread()
 		$this->_grabber->setEarliestMsg($context['showing_all_topics'] ? earliest_msg() : 0);
 
 		// @todo Add modified_time in for log_time check?
-		// Let's copy things out of the log_topics table, to reduce searching.
-		if ($modSettings['totalMessages'] > 100000 && $context['showing_all_topics'])
-		{
-			$this->_grabber->createTempTable();
-		}
-
-		// All unread replies with temp table
-		if ($context['showing_all_topics'] && $this->_grabber->hasTempTable())
-		{
-			$this->_num_topics = $this->_grabber->numUnreads(false);
-			$type = 'message';
-		}
-		// New posts with or without temp table
-		elseif ($this->_is_topics)
```

Comment:
Is the removal of this intended?

Reply:
I'll double check.   It would have been part of https://github.com/elkarte/Elkarte/commit/9eff7c2e21325a3096ef63f19579ef6072727d45 where all the temp tables were removed or git could have gone wild.

Reply:
It happened so long ago I completely forgot! :facepalm: